contracts/Migrations.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Why difference licenses for files? This contradicts LICENSE in the main repo btw.

Reply:
Because it was auto-generated like that 🤷. You can see with
```sh
npm install --save truffle
truffle init
cat contracts/Migrations.sol
```

Reply:
This should probably be changed then otherwise this may be unintentionally released under an MIT license

Reply:
Okay, removed the migrations files in 115be04.